codegen/package.go

```diff
@@ -212,6 +232,11 @@ func (p PackageHelper) MiddlewareSpecs() map[string]*MiddlewareSpec {
 	return p.middlewareSpecs
 }
 
+// MandatoryMiddlewareSpecs returns a map of middlewares available
```

Comment:
The comment of this method should be reworded to distinguish from `MiddlewareSpecs` method